src/components/dialogs/parameters/shortcircuit/short-circuit-parameters-utils.tsx

```diff
@@ -40,3 +51,21 @@ export const getStatus = (status: STATUS, styles: any) => {
     const color = status === STATUS.SUCCESS ? styles.succeed : styles.fail;
     return <Lens fontSize={'medium'} sx={color} />;
 };
+
+export const formatShortCircuitParameters = (
+    parameters: ShortCircuitParameters,
+    predefinedParameters: PREDEFINED_PARAMETERS
+): any => {
```

Comment:
RMQ => never use `any` type

Reply:
I didn't touch this line... i will put a TODO, there is an ongoing TS migration of those files.

Reply:
at least type this as Object